ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -138,3 +138,29 @@ def test_numpy_fftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device)
         n=n,
         d=d,
     )
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.fft2",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        shape=(2, 2),
+        array_api_dtypes=True
+    ),
+    s=st.sampled_from([(2, 2), (3, 3), (4, 4)]),
+    axes=st.sampled_from([(0, 1), (-2, -1)]),
+    norm=st.sampled_from([None, "backward", "ortho", "forward"]),
+)
+def test_numpy_fft2(dtype_and_x, s, axes, norm, frontend, test_flags, fn_tree, on_device):
```

Comment:
Please format this line into multiple lines so that it doesn't trigger this formatting warning `PEP 8: E501 line too long (90 > 88 characters)`